### ISSUE TITLE
A11y: feel editor has no aria label

### ISSUE THREAD

**Issue by pinussilvestrus** [bpmn-io/feel-editor#36]
### Describe the Bug

The feel editor does not have a screen reader readable label. E.g. when using a11y checker like `axe`, it throws an error

![image](https://user-images.githubusercontent.com/9433996/224282797-cd642432-d697-45b7-9c77-9a7202b81c3b.png)

### Steps to Reproduce

1. Render FEEL editor
2. Execute a11y checker, e.g. [axe dev tools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)
3. See a11y error is reported: https://dequeuniversity.com/rules/axe/4.6/aria-input-field-name?application=AxeChrome

### Expected Behavior

FEEL editor should have a screenreader readable label. One option could be to add an option to set the `aria-label`, `aria-labelledBy` or `title` attribute.

### Environment

- Host (Browser/Node version), if applicable: Chrome Version 111.0.5563.64 
- OS: macOS Ventura 13.2.1
- Library version: 0.7.0

**Comment by barmac** [comment]
Related to https://github.com/codemirror/dev/issues/1127

**Comment by barmac** [comment]
I just learnt that this **will not** be fixed if we allow to provide `id` since `label#for` cannot target contenteditable element (https://stackoverflow.com/questions/54792126/html-label-with-for-div-id-to-focus-a-contenteditable-div-does-label-label). So this needs to be one of the attributes from the issue.

**Comment by barmac** [comment]
See also https://github.com/codemirror/dev/issues/1140
